test_upstream_apis.sh

```diff
@@ -0,0 +1,122 @@
+#!/bin/bash
+
+#### SETUP ####
+
+# The directory this file is in must contain a file named 'sar_endpoint_config' which includes the following fields:
```

Comment:
I'd probably add a comment here saying "If secret keys contain $ symbols, make sure to escape it with \"